src/crypto/slow-hash.c

```diff
@@ -884,7 +901,9 @@ union cn_slow_hash_state
   b[0] = p[0]; b[1] = p[1]; \
   VARIANT2_PORTABLE_INTEGER_MATH(b, c); \
   __mul(); \
+  VARIANT2_2_1(); \
   VARIANT2_SHUFFLE_ADD_NEON(hp_state, j); \
+  VARIANT2_2_2(); \
```

Comment:
I remember there were two implementations for ARM, you updated only one of them.